public/app/plugins/datasource/azuremonitor/components/AzureCredentials.ts

```diff
@@ -0,0 +1,56 @@
+import { SelectableValue } from '@grafana/data';
+
+export enum AzureCloud {
+  Public = 'AzureCloud',
+  China = 'AzureChinaCloud',
+  USGovernment = 'AzureUSGovernment',
+}
+
+export const KnownAzureClouds = [
+  { value: AzureCloud.Public, label: 'Azure' },
+  { value: AzureCloud.China, label: 'Azure China' },
+  { value: AzureCloud.USGovernment, label: 'Azure US Government' },
+] as SelectableValue[];
```

Comment:
This type assertion is unnecessary.

Reply:
Changed to as it's defined in other datasources.

It will be gone with this PR: https://github.com/grafana/grafana/pull/80187